src/components/OfflineIndicator.js

```diff
@@ -14,33 +15,41 @@ const propTypes = {
     /** Information about the network */
     network: networkPropTypes.isRequired,
 
+    /** Styles for container element */
+    containerStyles: PropTypes.arrayOf(PropTypes.object),
+
     ...withLocalizePropTypes,
 };
 
+const defaultProps = {
+    containerStyles: [],
+};
+
 const OfflineIndicator = (props) => {
     if (!props.network.isOffline) {
         return null;
     }
 
     return (
         <View style={[
-            styles.chatItemComposeSecondaryRowOffset,
+            ...props.containerStyles,
             styles.flexRow,
             styles.alignItemsCenter]}
         >
             <Icon
-                src={Expensicons.Offline}
+                src={Expensicons.OfflineCloud}
                 width={variables.iconSizeExtraSmall}
```

Comment:
@MariaHCD Just wanted to confirm, this icon should be shown at 16x16. I think that's the case from seeing the mockups, but just double checking!

Reply:
It was set at 12x12 but I've updated to 16x16, thanks!

Reply:
![Screen Shot 2022-07-21 at 12 14 19 PM](https://user-images.githubusercontent.com/12268372/180165371-5ebda572-9300-487d-9401-9000e68ba39b.png)

![Screen Shot 2022-07-21 at 12 14 41 PM](https://user-images.githubusercontent.com/12268372/180165362-e9ba6e6a-fa6f-4b27-baae-9de4ee6a8da4.png)